crash_similarity.py

```diff
@@ -86,31 +110,36 @@ def get_stack_trace_for_uuid(uuid):
 
 
 def train_model(corpus):
-    if os.path.exists('stack_traces_model.pickle'):
-        return gensim.models.Doc2Vec.load('stack_traces_model.pickle')
+    
+    if checkif_model_trained_in_last_24hours()==True:
```

Comment:
why not  `if checkif_model_trained_in_last_24hours():` ?
it's better to move this condition outside train function. What if we download new data and want to retrain our model?